runners/flink/runner/src/main/java/org/apache/beam/runners/flink/FlinkRunner.java

```diff
@@ -340,6 +343,9 @@ protected String getKindString() {
    * org.apache.beam.sdk.transforms.View.AsMultimap View.AsMultimap} for the
    * Flink runner in streaming mode.
    */
+  @edu.umd.cs.findbugs.annotations.SuppressFBWarnings(
+      value = "SE_BAD_FIELD",
+      justification = "PTransform is not really Serializable, see note on PTransform. ")
```

Comment:
you can almost certainly just make `runner` `transient`, but this is fine too :)